include/aos/common/cryptoutils.hpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CRYPTOUTILS_HPP_
+#define AOS_CRYPTOUTILS_HPP_
+
+#include "aos/common/crypto.hpp"
+#include "aos/common/pkcs11/pkcs11.hpp"
+#include "aos/common/uuid.hpp"
+
+namespace aos {
+namespace cryptoutils {
+
+class Helper {
```

Comment:
Helper is too generic name. Rename it to `CertLoader` for example.

Reply:
renames to CertLoader